lib/assessment.js

```diff
@@ -5,7 +5,13 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
-
+  var minimum = arguments[0];
+  for (var i = 0; i < arguments.length; i++){
+    if(minimum < arguments[i]){
+      minimum = arguments[i];
+    }
+  }
+  return minimum; //i think this is right, but can't seem to get it to work
```

Comment:
You've got the sign backwards in line 10 - if a number becomes the new minimum by being _less_ than the current minimum. Otherwise, everything is right!